README.md

```diff
@@ -1,2 +1,20 @@
 # docker-visualization-xbeach
 Docker container to visualize xbeach model results.
+
+## Run docker-compose example:
+Run de following commands to start te docker compose example:
```

Comment:
Zitten wat spelfouten in de readme. Wilde het zelf verandere, maar lukte niet helemaal :)
 
+Run **the** following commands to start **the** docker compose example: